lib/teamdynamix_api.rb

```diff
@@ -1,20 +1,140 @@
 require 'net/http'
-class TeamDynamixApi
+class TeamdynamixApi
   if SETTINGS[:teamdynamix].blank?
     raise('Missing configurations for the plugin see https://github.com/MiamiOH/foreman_teamdynamix')
   end
-  if SETTINGS[:teamdynamix][:appID].blank?
-    raise('Missing AppID in plugin settings')
+  API_CONFIG = SETTINGS[:teamdynamix][:api]
+
+  if API_CONFIG[:id].blank?
+    raise('Missing Team Dynamix AppID in plugin settings')
```

Comment:
Missing Team Dynamix api id in plugin settings

Reply:
corrected